### PR TITLE
[CMAKE] Update CMake for CUDA compat

### PR DESCRIPTION
**Pull request by larshg**

### CODE REVIEW

File: cmake/pcl_find_cuda.cmake [hunk 1]
```diff
@@ -24,16 +24,12 @@ if(CUDA_FOUND)
 
   # Find a complete list for CUDA compute capabilities at http://developer.nvidia.com/cuda-gpus
 
-  if(NOT ${CUDA_VERSION_STRING} VERSION_LESS "10.0")
+  if(NOT ${CUDA_VERSION_STRING} VERSION_LESS "11.0")
```
Comment (SergioRAgostinho) [inline cmake/pcl_find_cuda.cmake:27]: Could you post a URL or even better add comment on where to check the appropriate `__cuda_arch_bin` for each toolkit version?

**Comment by kunaltyagi** [comment]
@SergioRAgostinho Milestone this please. 1.11.1 or 1.12.0

File: cmake/pcl_find_cuda.cmake (hunk 1 above)
Comment (larshg) [reply cmake/pcl_find_cuda.cmake:33]: I have added the comment.

**Comment by SergioRAgostinho** [comment]
Milestoning for 1.11.1 because there's code in the repo already using _shfl_sync instructions already. If not for that it would be 1.12.0.

**Pull request merged**